dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops

```diff
@@ -220,3 +220,20 @@ warning.could.not.expand.application.useraction=Attempt to resolve the issue in
 error.file.outside.archive=CWWKZ0132W: The file {0} in the archive {1} will not be extracted because the path is not contained within the archive.
 error.file.outside.archive.explanation=The file's path specifies a relative directory. Extracting the file would result in the file being located outside of the archive's root directory.
 error.file.outside.archive.useraction=Remove any files with relative paths from the archive.  
+
+#{0} is the application name, {1} is the location of the EAR or WAR file, {2} is the location where the file is expanded
+info.expanding.app=CWWKZ0133I: The application named {0} at location {1} is being expanded to the directory {2}.
+info.expanding.app.explanation=The archive file for the application is being expanded because automatic expansion of applications was configured.
+info.expanding.app.useraction=No action is required.
+
+info.loose.app=CWWKZ0134I: The application named {0} is using the loose application definition at location {1}.
+info.loose.app.explanation=The location for the application points to an XML file that defines a loose application. 
+info.loose.app.useraction=No action is required.
+
+info.directory.app=CWWKZ0135I: The application named {0} is using the expanded directory at location {1}.
+info.directory.app.explanation=The location for the application points to a directory rather than an archive file. 
+info.directory.app.useraction=No action is required.
+
+info.unexpanded.app=CWWKZ0136I: The application named {0} is using the archive file at location {1}.
```

Comment:
**change**
The application named {0} is using the archive file at location {1}.
**to**
The {0} application is using the archive file at the {1} location.